gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -454,14 +455,14 @@ private int readFooterContentIntoBuffer(ByteBuffer byteBuffer) {
     return bytesToWrite;
   }
 
-  private void requestObjectMedia(@Nullable Integer bytesToRead) throws IOException {
+  private void requestObjectMedia(long bytesToRead) throws IOException {
     GetObjectMediaRequest.Builder requestBuilder =
         GetObjectMediaRequest.newBuilder()
             .setBucket(resourceId.getBucketName())
             .setObject(resourceId.getObjectName())
             .setGeneration(objectGeneration)
             .setReadOffset(position);
-    if (bytesToRead != null) {
+    if (bytesToRead != 0) {
```

Comment:
If `bytesToRead` may not have a value, please change the type to `OptionalLong` (and same to the return type of `getBytesToRead()`